### PR TITLE
Add Support For Newer Versions of Dependencies

### PR DESCRIPTION
**Pull request by jcbantuelle**
Long time no talk Gabe! I hope you've been well and that fatherhood is suiting you.

I've been in the process of upgrading our favorite Tuition application from Rails 3/Ruby 2.0 up to contemporary versions and ran into deprecation warnings on this gem when I hit Ruby 2.4.

This PR adds Docker support for local testing, and tests against the latest versions of `attr_encrypted`, `activesupport`, and `activerecord`, as well as confirming the tests still pass on Ruby 2.7 and Postgres 11.

The only actual code change to the encryptor was removing ActiveSupport's deprecated `alias_method_chain`, otherwise it was just making sure things still worked on the latest versions which seems to be the case. Not too surprising given how lean this gem is overall.

Please let me know if you have any concerns or feedback.

### CODE REVIEW

File: lib/attr_encrypted_pgcrypto/log_subscriber/postgres_pgp.rb [hunk 1]
```diff
@@ -9,7 +9,8 @@ module PostgresPgp
       extend ActiveSupport::Concern
 
       included do
-        alias_method_chain :sql, :postgres_pgp
+        alias_method :sql_without_postgres_pgp, :sql
+        alias_method :sql, :sql_with_postgres_pgp
```
Comment (gabetax) [inline lib/attr_encrypted_pgcrypto/log_subscriber/postgres_pgp.rb:13]: Module#prepend could be an option here too

File: app_container_name [hunk 2]
```diff
@@ -0,0 +1 @@
+attr_encrypted_pgcrypto
```
Comment (gabetax) [inline app_container_name:1]: Enable trailing new lines in your editor

File: lib/attr_encrypted_pgcrypto/log_subscriber/postgres_pgp.rb (hunk 1 above)
Comment (jcbantuelle) [reply lib/attr_encrypted_pgcrypto/log_subscriber/postgres_pgp.rb]: Great suggestion, thanks! That cleaned up the code substantially.

**Comment by jcbantuelle** [comment]
Thanks for making me a contributor! I'd be happy to handle the rubygems release as well, my email is jcbantuelle@gmail.com on there (username is jcbantuelle)

**Pull request merged**